term_bsd.go

```diff
@@ -7,7 +7,7 @@ import "syscall"
 type attr syscall.Termios
 
 func (a *attr) setSpeed(baud int) error {
-	var rate uint64
+	var rate uint32
```

Comment:
Without this, I get:

swills/term/term_bsd.go:51: cannot use syscall.CS8 | syscall.CREAD | syscall.CLOCAL | rate (type uint64) as type uint32 in assignment
swills/term/term_bsd.go:52: cannot use rate (type uint64) as type uint32 in assignment
swills/term/term_bsd.go:53: cannot use rate (type uint64) as type uint32 in assignment
FAIL    github.com/swills/term [build failed]

Don't know if this is needed on other OSs, but can update if needed.